config/routes.rb

```diff
@@ -9,7 +9,9 @@
   resources :donations, only: [:index, :show]
   resources :cart#, only: [:index, :create, :update]
 
-  resources :users, only: [:new, :create, :show]
+  resources :users, only: [:new, :create]
+
+  resources :orders, only: [:index]
 
```

Comment:
@pixapi We probably shouldn't have added Order routes this until we actually needed them. Instead, maybe we could've used a placeholder button or just commented out that line in the test until it's fully functional?